services/user.py

```diff
@@ -0,0 +1,49 @@
+from django.contrib.auth.hashers import make_password
+
+from db.models import User

```

Comment:
Consider using `get_user_model()` instead of directly importing `User` to ensure compatibility with custom user models.